### PR TITLE
[Enchancement](Agg State)  storage function name and result is nullable in agg state type 

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes

storage function name and result is nullable in agg state type

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run arm

**Review by zhangstar333 — APPROVED** [review]

File: gensrc/thrift/Descriptors.thrift [hunk 1]
```diff
@@ -38,6 +38,7 @@ struct TColumn {
     14: optional i32 gram_size
     15: optional i32 gram_bf_size
     16: optional string aggregation
+    17: optional bool result_is_nullable
```
Comment (HappenLee) [inline gensrc/thrift/Descriptors.thrift:41]: why not use the is allow null ？ if add a new property add comment with the difference

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**